webpack.release.config.js

```diff
@@ -46,22 +37,9 @@ var cdnConfig = config('okta-sign-in.min.js');
 cdnConfig.entry.unshift('babel-polyfill');
 cdnConfig.plugins = plugins({ isProduction: true, analyzerFile: 'okta-sign-in.min.analyzer' });
 
-// 3. noJqueryConfig
-var noJqueryConfig = config('okta-sign-in-no-jquery.js');
```

Comment:
is this considered a "breaking" change? Since we are not producing an `okta-sign-in-no-jquery.js` anymore... might want to check with PM if this means we need some extra announcement or if we need some more work about this.

Reply:
yeah, as I mentioned in the description and certainly will double check with PM.